### PR TITLE
New version: lin-ycv.EverythingPowerToys version 0.78.0

### PR DESCRIPTION
**Pull request by lin-ycv**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137326)

### CODE REVIEW

**Comment by lin-ycv** [comment]
the dependency is in review:
https://github.com/microsoft/winget-pkgs/pull/137204

**Comment by lin-ycv** [comment]
everything seems correct, not sure why the bot cant find the dependency
```
Manifest has the following dependencies that were not validated; ensure that they are valid:
  - Packages
      Microsoft.PowerToys [>= 0.78.0]
Manifest validation succeeded.
```
```
Name                Id                  Version Source
-------------------------------------------------------
PowerToys (Preview) Microsoft.PowerToys 0.78.0  winget
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-01T18:31:18.9527121Z ##[error] One or more errors occurred. (Manifest Error: No Suitable Minimum Version:  [PackageIdentifier] Value: Microsoft.PowerToys

(Automated response - build 695.)

File: manifests/l/lin-ycv/EverythingPowerToys/0.78.0/lin-ycv.EverythingPowerToys.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.PowerToys
+    MinimumVersion: 0.78.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (CoolPlayLin) [inline manifests/l/lin-ycv/EverythingPowerToys/0.78.0/lin-ycv.EverythingPowerToys.installer.yaml:17]:
```suggestion
```
这一段没必要

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
@stephengillie https://github.com/microsoft/winget-pkgs/labels/Validation-No-Executables should be waived.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**